app/views/una_chapters_and_book_sections/edit_fields/_date_published.html.erb

```diff
@@ -0,0 +1,4 @@
+<%# see app/views/records/edit_fields/_date_published.html.erb %>
+<% template = f.object.model.class.to_s.underscore %>
+
+<%= render "una/date_fields", date_type: 'published', template: template, f: f %>
```

Comment:
You're using single quotes here and double quotes in `app/views/una_books/edit_fields/_date_published.html.erb`. 

There are a few other places where single quotes are being used. Can we only use double quotes please, unless there is additional functionality required within a string interpolation.

Reply:
two types of quotes are being used in the same line, first its `render "una/date_fields"`, then `date_type: 'published'`